src/loaders/express.js

```diff
@@ -76,9 +76,23 @@ export default (app) => {
     }
 
     // Adding headers (CORS)
-    app.use((_, res, next) => {
-        // Allow connections for all origins
+    app.use((req, res, next) => {
+        // Allow connections for default specified connections
         res.setHeader("Access-Control-Allow-Origin", config.access_control_allow_origin);
+
+        // Allow requests from connections specified by the allow list regexes
+        const origin = req.header("origin")?.toLowerCase();
+        if (origin) {
+            const originAllowed = config.access_control_allow_origin_regex_list.find((allowOrigin) => {
+                const matches = origin.match(RegExp(allowOrigin, "g"));
```

Comment:
There's probably a built-in way to just check if a particular string matches a regex, no need to get the matches, maybe .test or .matches or something, can't remember.